Components/Image/image.css

```diff
@@ -0,0 +1,29 @@
+/* * {
+  outline: 1px solid red;
+} */
+
+.image-container {
+  display: flex;
+  height: 10rem;
+  width: 50rem;
+  flex-direction: row;

```

Comment:
flex direction row is by default, you don't have write that

Reply:
Removed that property from that class